### ISSUE TITLE
The header for the generated file is duplicated

### ISSUE THREAD

**Issue by AdrianoRuberto** [Manweill/swagger-axios-codegen#140]
For the following code:
```js
const { codegen } = required("swagger-axios-codegen")

codegen({
   methodNameMode: "operationId",
   remoteUrl: "....",
   outputDir: "....",
});
```
I get the this:
![image](https://user-images.githubusercontent.com/17402643/125289338-6dc43a80-e31f-11eb-94b8-5db6ce32b3a3.png)

Also it's better to use `@ts-nocheck` instead of `@ts-ignore` because `@ts-ignore` will only work for the line below it, but `@ts-nocheck` will disable the checking for the entire file.

**Comment by Manweill** [comment]
Welcome PR

**Comment by Manweill** [comment]
fix this issue in v0.12.5
